### PR TITLE
added Hyundai Veloster 2020 FPv2

### PR DESCRIPTION
**Pull request by nepro012**
Fingerprint for 2020 Veloster(Korea) fingerprint v2.0.

### CODE REVIEW

File: selfdrive/car/hyundai/values.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ class CAR:
   SONATA = "HYUNDAI SONATA 2020"
   SONATA_LF = "HYUNDAI SONATA 2019"
   PALISADE = "HYUNDAI PALISADE 2020"
-  VELOSTER = "HYUNDAI VELOSTER 2019"
+  VELOSTER = "HYUNDAI VELOSTER 2019-2020"
```
Comment (VirtuallyChris) [inline selfdrive/car/hyundai/values.py:36]: Revert this change, only update the model year in the readme

**Review by VirtuallyChris — DISMISSED** [review]

**Comment by pd0wm** [comment]
What's your dongle id?

File: selfdrive/car/hyundai/values.py (hunk 1 above)
Comment (nepro012) [reply selfdrive/car/hyundai/values.py]: Done!

**Comment by nepro012** [comment]
> What's your dongle id?

it's 8a1ed84bfc7d0c84

**Review by pd0wm — APPROVED** [review]

**Comment by nepro012** [comment]
> What harness do you use?
It's Hyundai E from comma.ai + Comma power
